.github/workflows/tag-release.yml

```diff
@@ -36,5 +36,8 @@ jobs:
       id: draft_release
```

Comment:
Is it still a draft if publish is set to true?

Reply:
good point! changed